source/includes/fact-connect-intro.rst

```diff
@@ -0,0 +1,7 @@
+The :ref:`mongosync <c2c-mongosync>` utility creates a connection
+between MongoDB clusters. ``mongosync`` can make a connection between:
+
+- :ref:`self-managed clusters <server-replica-set-deploy-all>`,
```

Comment:
is it correct to have the "," at the end of this bullet?

Reply:
Yes, it's correct. The comma is there because we want to say that mongosync can make connections within these three categories. Without the comma we would be saying mongosync can make connections between the categories. 

In other words we want to say

hosted to hosted
hosted to atlas
atlas to atlas

not 

(hosted to hosted) to ( (hosted to atlas) or (atlas to atlas) ) 

etc.